README.md

```diff
@@ -22,20 +22,28 @@ For a more detailed view of how we think about Lyra, check out our introductory
 
 ## Getting Started
 
+### Build
 The project requires [Go](https://golang.org/doc/install) 1.11 or higher, and [go modules](https://github.com/golang/go/wiki/Modules) to be on.
 
 1. Clone the git repo: `$ git clone https://github.com/lyraproj/lyra`
 2. Build the binary: `$ cd lyra; make lyra`
-3. Run the binary with the sample Workflow: ` $ ./build/lyra --debug apply sample`
 
-You can view the sample Workflow that is applied by the command above in `plugins/example.pp`.  This does not touch any real-world resources.
+### Deploying Workflows with CLI
 
-There is an AWS Workflow called `attach` in `plugins\attach.pp` (To run: ` $ ./build/lyra --debug apply attach`). This will use the default AWS region configured in `~/aws/config` and create real AWS resources.  To delete the Workflow (i.e. its resource), run ` $ ./build/lyra --debug delete attach`.  The `data.yaml` file can be used to configure AWS tag data.
+> **!! WARNING: THIS WORKFLOW CREATES REAL RESOURCES ($$) !!**
 
-## Integrating with Kubernetes
-1. Install the Lyra CRD as described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
-2. Start Lyra in controller mode: ` $ ./build/lyra --debug controller`
-3. Create a Workflow resource, also described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
+1. Run the binary with the [sample Workflow](https://github.com/lyraproj/lyra/blob/master/plugins/attachyaml.yaml): ` $ ./build/lyra apply awsyaml --debug`
+2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete attach --debug`.  
+
+This workflow is an AWS Workflow called `attachyaml` in `plugins\attachyaml.yaml`. This will use the default AWS credentials configured in your `~/.aws/config`. Region is currently hard-coded to `eu-west-1`. 
+
+
+### Deploying Workflows with Kubernetes
```

Comment:
Needs something about building and running the operator

Reply:
Operator should be auto pulled into the build: 

https://github.com/lyraproj/lyra/blob/master/cmd/lyra/cmd/controller.go#L12

I don't think there's anything else needed @scottyw?

Reply:
It's opposite. The controller works in a pull model from the cluster. The controller subscribes to 'Workflow' events created on the cluster. When starting the controller, it pulls the same kubeconfig that `kubectl` uses to figure out which k8s cluster to start listening to. Registering the CRD is what allows the cluster to understand what a 'Workflow' is. 

It's a pretty brilliant model the K8s community came up with that allows you to remotely dev your controller, but subscribe to a "production" k8s cluster. It's also what I'm noodling on in this proposal:
https://github.com/kenazk/lyra/blob/proposal/docs/lyra-controller-proposal/proposal.md